tensorflow/python/summary/writer/event_file_writer.py

```diff
@@ -108,7 +112,9 @@ def close(self):
 
     Call this method when you do not need the summary writer anymore.
     """
+    self._worker.stop()
     self.flush()
+    self._worker.join()
```

Comment:
@mrry I should not use while True: continue because this will consume much cpu resource.